packages/financial-templates-lib/src/price-feed/CryptoWatchPriceFeed.js

```diff
@@ -145,6 +145,10 @@ class CryptoWatchPriceFeed extends PriceFeedInterface {
       });
   }
 
+  getEarliestTime() {
+    return this.earliestTime;
```

Comment:
Similar to below, should this be the _expected_ earliest time based on the price feed construction? Or should it be the earliest time we have data for (which could be now if historical data is lacking or unavailable because of a feed error)?